include/common/qrack_types.hpp

```diff
@@ -0,0 +1,24 @@
+#define bitLenInt uint8_t
```

Comment:
Needs header boilerplate and `#pragma once`.

Reply:
Done.